queue_job/tests/test_runner_channels.py

```diff
@@ -1,10 +1,43 @@
 # Copyright 2015-2016 Camptocamp SA
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
 
+from odoo.tests import BaseCase
+
 # pylint: disable=odoo-addons-relative-import
 # we are testing, we want to test as we were an external consumer of the API
 from odoo.addons.queue_job.jobrunner import channels
 
 from .common import load_doctests
 
 load_tests = load_doctests(channels)
+
+
+class TestChannelManager(BaseCase):
+    def test_subcapacity_default(self):
+        cm = channels.ChannelManager()
+        cm.simple_configure("root:4")
+        root = cm.get_channel_by_name("root")
+        self.assertEqual(root.capacity, 4)
+        self.assertEqual(root.subcapacity, 0)
```

Comment:
What is the reason for the default subcapacity to be 0 instead of None?

Reply:
Personal preference for subcapacity value to be contained by the set of whole numbers rather than the set of counting numbers plus None.

Reply:
But the default for capacity is `None` so I'd do the same for subcapacity, if only for consistency.
Also, I think we could say there is a distinction between a capacity of 0 (a channel that is paused?) vs an undefined capacity.